content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -162,7 +166,11 @@ $ source ~/istioctl.bash
 
 Installing the ZSH auto-completion file
 
-For ZSH users, the `istioctl` auto-completion file is located in the `tools` directory. Copy the `_istioctl` file to your home directory, or any directory of your choosing (update directory in script snippet below), and source the `istioctl` auto-completion file in your `.zshrc` file as follows:
+For ZSH users, the `_istioctl` auto-completion file is located in the `tools` directory.
+
+You can also generate it by running `istioctl collateral completion --zsh -o .`.
+
+Copy the `_istioctl` file to your home directory, or any directory of your choosing (update directory in script snippet below), and source the `_istioctl` auto-completion file in your `.zshrc` file as follows:
```

Comment:
I don't think the normal way to add a completion file is to source it with .zsh. Normally it would be placed so it was found in FPATH or in a folder on FPATH..

Reply:
To be honest, I don't have enough ZSH knowledge to hell. These were the previous instructions.